frontend/src/metabase/dashboard/types/embed-display-options.ts

```diff
@@ -29,7 +29,7 @@ export type EmbedDisplayParams = {
   hideParameters: EmbedHideParameters;
   font: EmbedFont;
   theme: DisplayTheme;
-  getClickActionMode?: ClickActionModeGetter;
+  getClickActionMode?: ClickActionModeGetter | undefined;
```

Comment:
nit: `?` makes it `| undefined` already

```suggestion
  getClickActionMode?: ClickActionModeGetter;
```